configure.ac

```diff
@@ -1247,27 +1251,13 @@ else
 fi
 AM_CONDITIONAL(BGPD, test "x$BGPD" = "xbgpd")
 
-if test "${enable_ripd}" = "no";then
-  RIPD=""
-else
-  RIPD="ripd"
-fi
-AM_CONDITIONAL(RIPD, test "x$RIPD" = "xripd")
+AM_CONDITIONAL(RIPD, test "${enable_ripd}" != "no")
```

Comment:
AM_CONDITIONAL will make RIPD available if it is enabled but doesn't it just set to whatever it was set to during the configuration? so "yes" is a possible value for example. Am I missing something here?